Reapply "Delete TorchScript based Android demo app and point to ExecuTorch (#153633)"

larryliu0820 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #153656

This reverts commit ae0e8f0c7316addab3f415dc767a9d34f58b0dae.

Keep android/libs/fbjni because it's being used by other components of
PyTorch.

cc @albanD

larryliu0820 committed:
Reapply "Delete TorchScript based Android demo app and point to ExecuTorch (#153633)"

This reverts commit ae0e8f0c7316addab3f415dc767a9d34f58b0dae.

Keep android/libs/fbjni because it's being used by other components of
PyTorch.

[ghstack-poisoned]

malfet reviewed (approved):
LGTM, though please close original PR then

larryliu0820 committed:
Update on "Reapply "Delete TorchScript based Android demo app and point to ExecuTorch (#153633)""


This reverts commit ae0e8f0c7316addab3f415dc767a9d34f58b0dae.

Keep android/libs/fbjni because it's being used by other components of
PyTorch.

cc albanD

[ghstack-poisoned]

larryliu0820 committed:
Update on "Reapply "Delete TorchScript based Android demo app and point to ExecuTorch (#153633)""


This reverts commit ae0e8f0c7316addab3f415dc767a9d34f58b0dae.

Keep android/libs/fbjni because it's being used by other components of
PyTorch.

cc albanD

[ghstack-poisoned]

larryliu0820 commented:
@pytorchbot merge

larryliu0820 commented:
@pytorchbot merge

larryliu0820 commented:
> Thanks for getting this through! Curious where is fbjni being used?


😳oh no I think its not being used and I forgot to remove it. This was the broken ci in previous pr: https://github.com/pytorch/pytorch/actions/runs/15052773937/job/42311383050 

I removed the option BUILD_JNI in this pr but forgot to remove fbjni. I can add a follow up.

larryliu0820 commented:
ok this is being used internally, in production so can't delete.

larryliu0820 commented:
@pytorchbot revert

larryliu0820 commented:
@pytorchbot revert -m "Still being used internally so can't remove" -c ignoredsignal

jeanschmidt commented:
@pytorchbot revert -m "Broke lots of things internally, @malfet may you help the author? [D74894758](https://www.internalfb.com/diff/D74894758)" -c ghfirst

jeanschmidt commented:
@pytorchbot revert -m "Broke lots of things internally, @malfet may you help the author? [D74894758](https://www.internalfb.com/diff/D74894758)" -c ghfirst

larryliu0820 commented:
> @pytorchbot revert -m "Broke lots of things internally, @malfet may you help the author? [D74894758](https://www.internalfb.com/diff/D74894758)" -c ghfirst

@jeanschmidt thank you for your attention. This is already reverted and I'm iterating on it.

malfet commented:
> @pytorchbot revert -m "Broke lots of things internally, @malfet may you help the author? [D74894758](https://www.internalfb.com/diff/D74894758)" -c ghfirst

@jeanschmidt @larryliu0820 is a metamate :)

jeanschmidt commented:
Ohh, why it was appearing closed to me? Anyways, thanks for the info :) 

and sorry for poking you @malfet :)

larryliu0820 commented:
Start a co-dev PR/diff, closing this.
